roles/kube-control-plane/templates/kubeadm.yml.j2

```diff
@@ -26,6 +26,8 @@ dns:
   imageRepository: {{ kubernetes_image_registry }}{{ coredns_image_prefix }}
 {% endif %}
 apiServer:
+  certSANs:
+{{ kubernetes_apiserver_certSANs | to_nice_yaml | indent(2, true) }}
```

Comment:
Have you tested what happens when you leave the default value for `kubernetes_apiserver_certSANs`?

Do you need to add an `if` clause or is an empty list ok here?

Reply:
Good catch @ralgozino.
The behavior doesn't change since those are extra names, but the indentation may be ugly to see, so I put an `if` on the template (ref: [c112829](https://github.com/sighupio/fury-kubernetes-on-premises/pull/80/commits/c112829bd7a530e95b20b8372a528b3f6e5e0bc6))